test/torchaudio_unittest/functional/autograd_impl.py

```diff
@@ -78,6 +78,47 @@ def test_lfilter_batching(self):
                           [0.7, 0.2, 0.6]])
         self.assert_grad(F.lfilter, (x, a, b))
 
+    def test_filtfilt_a(self):
+        torch.random.manual_seed(2434)
+        x = get_whitenoise(sample_rate=22050, duration=0.01, n_channels=2)
```

Comment:
For the GPU test error, I think something like `get_whitenoise(..., device=self.device, dtype=self.dtype)` would do.